bin/worker.php

```diff
@@ -47,16 +48,17 @@
 
 Addon::loadHooks();
 
-$spawn = (($_SERVER["argc"] == 2) && ($_SERVER["argv"][1] == "spawn"));
+$parentID = Argument::get('parent', $argv, null, 'integer');
```

Comment:
Maybe rename this to `$parentId` ?

Reply:
Hm .. I'm more a fan of short arguments, but I'm not dogmatic.
I've therefore to check the difference between `parentId` and `parentID` f.e.

Reply:
I mean I like the `camelCase` way. :-)

Reply:
Of course but you can account for argument casing mistakes.

Reply:
ah now I undertand @Quix0r  .. not `parentID` - it should be `parentId`
I thought I should rename the argument `parent` to `parentId`
=> fully understandable for me and I'll change it

Reply:
@nupplaphil fixed my comment.